roles/calico_master/tasks/main.yml

```diff
@@ -1,26 +1,34 @@
 ---
-- name: Calico Master | Assure the calico certs have been generated
-  stat:
-    path: "{{ item }}"
-  with_items:
-  - "{{ calico_etcd_ca_cert_file }}"
-  - "{{ calico_etcd_cert_file }}"
-  - "{{ calico_etcd_key_file }}"
+- include: certs.yml
 
-- name: Calico Master | Create temp directory for policy controller definition
+- name: Calico Master | oc adm policy add-scc-to-user privileged system:serviceaccount:kube-system:calico-node
+  oc_adm_policy_user:
+    user: system:serviceaccount:kube-system:calico-node
+    resource_kind: scc
+    resource_name: privileged
+    state: present
+
+- name: Calico Master | oc adm policy add-scc-to-user privileged system:serviceaccount:kube-system:calico-kube-controllers
```

Comment:
Why don't we do this in the upgrade playbook?

Reply:
added.